src/mumble/widgets/TrayIcon.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright 2024 The Mumble Developers. All rights reserved.
```

Comment:
The date should be removed now that we have removed it from all other copyright headers on master